extensions/ql-vscode/src/common/app.ts

```diff
@@ -18,6 +18,7 @@ export interface App {
   readonly workspaceState: Memento;
   readonly credentials: Credentials;
   readonly commands: AppCommandManager;
+  readonly queryServerCommands: QueryServerCommandManager;
```

Comment:
Should get a second opinion about exposing the `queryServerCommands` field on the app interface. @koesie10, do you have any thoughts? Is there a reason not to, or did we just not need it until this point?

Reply:
I think we just haven't needed it until this point. I don't see a particular reason not to expose it, it's coming from the same file so we aren't importing anything extra.